### ISSUE TITLE
Youtube links not resolved in v3.0.0

### ISSUE THREAD

**Issue by brecke** [lmammino/tall#17]
Hey there,

I tried upgrading from v2 to v3 and youtube links are not resolved anymore. You may try this example I took from the README:

```
import { tall } from 'tall';

async function someFunction() {
  try {
    const LINK = 'https://youtu.be/TCB_RSlgTqY';
    const unshortenedUrl = await tall(LINK, {
      maxRedirect: 10
    });
    console.log('Tall url', unshortenedUrl);
  } catch (error) {
    console.error('AAAW 👻', error);
  }
}

someFunction();
```

Am I doing something wrong? 

Thanks in advance,

**Comment by lmammino** [comment]
Hello @brecke, thanks for reporting this. It sounds like a bug. I won't have a lot of time in the next few weeks to investigate this. Would you like to contribute and submit a PR to resolve this issue?

**Comment by brecke** [comment]
Hi,

I would under normal circumstances but I really can’t, I am moving countries with two cats and a baby on the way in the middle of a global pandemic, too much to deal with right now 😵 

Maybe in a few months time, who knows..

**Comment by lmammino** [comment]
This should be fixed in 4.0.1 (just released) :)
